privacy_evaluator/classifiers/classifier.py

```diff
@@ -32,34 +33,43 @@ def predict(self, x: np.ndarray):
         """
         return self.art_classifier.predict(x)
 
+    def to_art_classifier(self):
+        """Converts the classifier to an ART classifier.
+
+        :return: Converted ART classifier.
+        """
+        return self.art_classifier
+
     @staticmethod
     def _to_art_classifier(
         classifier: Union[tf.keras.Model, torch.nn.Module],
+        loss: Union[tf.keras.losses.Loss, torch.nn.modules.loss._Loss],
         nb_classes: int,
         input_shape: Tuple[int, ...],
     ) -> Union[TensorFlowV2Classifier, PyTorchClassifier]:
-        """Converts a classifier to an ART classifier.
+        """Initializes an ART classifier.
 
-        :param classifier: Classifier to be converted. Either a Pytorch or Tensorflow classifier.
+        :param classifier: Original classifier, either Pytorch or Tensorflow.
         :param nb_classes: Number of classes that were used to train the classifier.
-        :param input_shape: Input shape of a data point of the classifier.
-        :return: Given classifier converted to an ART classifier.
-        :raises TypeError: If the given classifier is of an invalid type.
+        :param input_shape: Shape of a input data point of the classifier.
+        :return: Instance of an ART classifier.
+        :raises TypeError: If `classifier` is of invalid type.
         """
         if isinstance(classifier, torch.nn.Module):
             return PyTorchClassifier(
                 model=classifier,
-                loss=None,
+                loss=loss,
                 nb_classes=nb_classes,
                 input_shape=input_shape,
             )
         if isinstance(classifier, tf.keras.Model):
```

Comment:
keras is just one way to use tensorflow. You should, therefore, not check if it is a keras model, but if it is a TensorFlow model.